conan/tools/cmake/cmake.py

```diff
@@ -122,7 +122,7 @@ def _build(self, build_type=None, target=None, cli_args=None, build_tool_args=No
         if cmd_line_args:
             args += ['--'] + cmd_line_args
 
-        arg_list = [args_to_string([bf]), build_config, args_to_string(args)]
```

Comment:
Risk of 🍔 ?

Reply:
https://github.com/conan-io/conan/issues/11358